# Do not try to load ~/.LDM if it doesn't exist yet. Instead, reconf before loading.
--- lmd.orig	2011-07-03 17:19:24.000000000 -0300
+++ lmd	        2012-06-07 15:11:51.645346908 -0300
@@ -2,6 +2,8 @@
 #
 
 
+[ ! -d ~/.LMD ] && lmd-fu -reconf
+[ $? -ne 0 ] && exit 1
 source ~/.LMD/options;
 source /usr/share/lmd/langs/$LMLANG;
 #function ESCAS(){

